src/Providers/MiddlewareProvider.php

```diff
@@ -5,13 +5,14 @@
 
 
 use Choredo\Entities\Family;
+use Choredo\Middleware\FamilyHydrator;
 use Choredo\Middleware\MultiTenantFamilyHydrator;
-use Doctrine\ORM\EntityManager;
+use Doctrine\ORM\EntityManagerInterface;
 use League\Container\ServiceProvider\AbstractServiceProvider;
 
 class MiddlewareProvider extends AbstractServiceProvider
 {
-    protected $provides = [ MultiTenantFamilyHydrator::class ];
+    protected $provides = [MultiTenantFamilyHydrator::class, FamilyHydrator::class];
```

Comment:
This is gonna get large, can we wrap down onto the next line?

Reply:
:+1: